package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.15.3",
     "github": "^9.2.0",
     "mz": "2.6.0",
```

Comment:
In the conversation, you seemed like you were mostly concerned with the asynchronous nature of Node. Why not just use some of `fs`'s synchronous functions instead?